config/infection.json.dist

```diff
@@ -21,6 +21,48 @@
         }
     },
     "timeout": 10,
-    "minMsi": 98,
-    "minCoveredMsi": 100
+    "minMsi": 100,
+    "minCoveredMsi": 100,
+    "mutators": {
+        "@default": true,
+        "Break_": {
+            "ignore": [
+                "Psl\\Collection\\*Map::zip"
+            ]
+        },
+        "DecrementInteger": {
+            "ignore": [
+                "Psl\\DataStructure\\PriorityQueue::peek"
```

Comment:
This fallback to `?? 0` will never occur since we first check the array size.
This is a false positive:

https://github.com/azjezz/psl/blob/4affb62545080af0457f78a2bdcfc33cc4057f47/src/Psl/DataStructure/PriorityQueue.php#L53

I could solve it by doing an additional `invariant` check, but this seems slower for no real reason.